### PR TITLE
Create wind barb example

### PR DESCRIPTION
**Pull request by twsearle**
Create an example app of wind barb extension to bokeh.

### CODE REVIEW

**Comment by andrewgryan** [comment]
@twsearle Still a long way to go to replicate what was there originally, e.g. `_u[i]` support and `.define(...)` but we have satisifed the type checker for now

**Comment by twsearle** [comment]
I think this is the basic wind barb example pretty much ready to go. 

I was going to add an optional argument to set the scale of the wind barbs, however I can't work out how to get this working. 

I changed to Bokeh version 1.4 because that version includes the examples, but this may not be necessary.

Do you think it would be worth committing this as is for now?

**Comment by andrewgryan** [comment]
@twsearle I think it is a significant step forward on the way to wind barb support. I'll commit it on Monday as my brain is a little fried right now.

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-bokeh
+bokeh=1.4.0
```
Comment (andrewgryan) [inline requirements.txt]: @twsearle this might need to be taken out for now until we officially move to the latest version of Bokeh

Comment (twsearle) [reply requirements.txt]: done.

**Pull request merged**